src/components/sbb-link-list/sbb-link-list.scss

```diff
@@ -24,9 +24,12 @@ sbb-title {
   @include link-list--horizontal;
 }
 
```

Comment:
:host { display: block } is not needed

Reply:
To change that sbb title is hidden from normal user, but still visible to screen reader user.
```
@mixin link-list--horizontal {
  display: flex;
  flex-wrap: wrap;
  column-gap: var(--sbb-spacing-fixed-8x);

  sbb-title {
     @include screenReaderOnly;
  }
}
```